File: straw-portal/src/main/java/cn/tedu/straw/portal/domian/vo/MyInfo.java
package cn.tedu.straw.portal.domian.vo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @Description: 用户信息
 * @Author: ChenHaiBao
 * @CreateDate: 2020/4/10$ 22:47$
 * @Version: 1.0
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class MyInfo {
    //金币数量
    private  Integer goldCount;
    //回答问题的数量
    private  Integer answerCount;
    //提问题的数量
    private  Integer questionCount;
    //任务的数量
    private  Integer taskCount;
    //收藏数量
    private Integer collectCount;
}

File: resources/js/myinfo.js
const myinfo=new Vue({
    el:"#myInfoApp" ,
    data:{
        goldCount:0,
        answerCount:0,
        questionCount:0,
        taskCount:0,
        collectCount:0
    },
    methods:{
        loadMyInfo:function () {
            let _this=this;
           $.get("/personal/getMyInfo",function (result) {
               if(result.code==200){
                   let data=result.data;
                _this.goldCount=data.goldCount;
                _this.answerCount=data.answerCount;
                _this.questionCount=data.questionCount;
                _this.taskCount=data.taskCount;
                _this.collectCount=data.collectCount;
               }
           });
        }
    },
    created:function () {
        this.loadMyInfo();
    }
});